test/integration/patients/worklist/worklist.js

```diff
@@ -906,12 +902,13 @@ context('worklist page', function() {
         fx.included.push({
           id: '1',
           type: 'flows',
-          attributes: _.extend(_.sample(this.fxFlows), {
+          attributes: {
             name: 'Test Flow',
-          }),
-          relationships: {
-            state: { data: { id: '55555' } },
+            details: null,
```

Comment:
were the details, created_at, updated_at needed in all the tests?

Reply:
I would guess no. I added them to match what data was being returned before.

Kind of related, but there's a few of these tests were if I remove that specific `fx.include` data, the test still passes. I hadn't looked into the details as to why that `fx.include` was originally put there.